doc/README_windows.txt

```diff
@@ -1,4 +1,5 @@
-Bitcoin Unlimited 0.12.1b

```

Comment:
Should be renamed to `README_windows.md`

Reply:
I had the same thought at first, but a standard windows install can't open a .md file.

We don't want windows users looking around the installed files unable to open the README file for their OS.
So I suggest keeping as a .txt for now.

Reply:
fair enough.

Even though I don't think is complicated for a window use to associate the `.md` extension to a text editor.  

Having said that I don't have a strong opinion, I'm ok with keep the `.txt` extensions